src/components/PipelineResourcesModal/GitResourceFields.js

```diff
@@ -0,0 +1,34 @@
+/*
+Copyright 2019 The Tekton Authors
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+import React from 'react';
+import { TextInput } from 'carbon-components-react';
+
+const GitResourceFields = props => {
+  const { revision, handleChangeTextInput, invalidFields } = props;
+
+  return (
+    <TextInput
+      id="revision"
+      placeholder="pipeline-resource-revision"
+      value={revision}
+      labelText="Revision"
```

Comment:
we need to use react-intl for user-facing strings

same for UniversalFields.js and PipelineResource.js

Reply:
ok, fixed this now :)

Reply:
still seeing hard-coded string here `labelText="Revision"`

two more `labelText` props below:
- https://github.com/tektoncd/dashboard/pull/942/files#diff-58230c181d9089fb7a4c65e207282174R43
- https://github.com/tektoncd/dashboard/pull/942/files#diff-58230c181d9089fb7a4c65e207282174R89

Reply:
oops, changed that now